ibis/backends/tests/conftest.py

```diff
@@ -1,9 +1,9 @@
 import importlib
 import os
-from pathlib import Path
 from typing import List
 
 import pandas as pd
+import pkg_resources
```

Comment:
so we have setup tools in the requirements?

Reply:
Good point. Just checked and we don't. We're already using `pkg_resources`, so I guess we're getting setuptools from a dependency.

I added setuptools to our dependencies (to `requirements.txt` which is read by `setup.py`).